Chat/AppManager.cpp

```diff
@@ -3,60 +3,59 @@
 #include <iostream>
 #include <sstream>
 
-DLL_EXP AppManager app_man;
+DLL AppManager app_man;
 
 AppManager::~AppManager(){}
 
-std::vector<UserMsg> AppManager::GetCurrentChat()
+const std::vector<UserMsg>* AppManager::GetPrivateChatMsgs(const std::string &name)const
 {
-    return chat_.GetCurrentChat();
+    return chat_.GetChatMsgs(name);
 }
 
-AppManager::AppManager()
+AppManager::AppManager(unsigned int broadc_port, unsigned int tcp_port)
 {
     chat_.SetNetwork(&network_);
     network_.SetChat(&chat_);
     chat_.SetFM(&fm_);
     network_.SetFM(&fm_);
-    network_.PrepareNetwork();
-    chat_.SetUserInfo(6, "Anton");
+    network_.PrepareNetwork(broadc_port, tcp_port);
 }
 
-void AppManager::SendFile(const std::string & path, const std::string & ip, const std::string & name)
-{
-    network_.SendFile(path, ip, name);
-}
-
-const std::string AppManager::GetFilePath(int file_index)const
+const std::string& AppManager::GetFilePath(int file_index)const
 {
     return fm_.GetFilePath(file_index);
 }
 
-const std::string AppManager::GetFileName(int file_index)const
+const std::string& AppManager::GetFileName(int file_index)const
 {
     return fm_.GetFileName(file_index);
 }
 
-void AppManager::SendList(const std::string & ip)
+void AppManager::SendList(const std::string& ip, unsigned int port)
```

Comment:
If we are worried about file sending (if it was sent successful or not), please, make this function boolean.